spec/services/support_interface/work_history_break_export_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::WorkHistoryBreakExport do
+  describe '#data_for_export' do
+    it 'returns an array of hashes for candidates with breaks in their work history' do
+      create(:candidate)
```

Comment:
The setup is quite noisy, and it takes a bit of effort to figure out what this example tests.
To move the focus on what is tested, consider using a before block to setup data that is required but you don't reference, and `let()` (or `let!()`to setup data that is references in the spec.

Reply:
Thanks for the idea Despo, I've started refactoring the tests and have pushed a WIP for this

Reply:
I've tidied that up now and it's ready for re-review. Hopefully, it's more readable 🤞🏽 let me know what you think